patterns/1-initial/innersource-hackathon.md

```diff
@@ -0,0 +1,80 @@
+## Title
+
+InnerSource Hackathon
+
+## Patlet
+
+In a company, initially only InnerSource enthusiasts are interested and practicing InnerSource during the early stages of InnerSource adoption; not all engineering teams are willing or have enough time and resources to adopt InnerSource. In this scenario, it is good to provide a safe space to try and adopt InnerSource through an InnerSource Hackathon event within the company.
```

Comment:
I love this pattern because it ties into the [crossing the chasm](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/1-initial/crossing-chasm.md) pattern: It's a lovely way to get people to practice instead of only learn or hear about best practices.